roles/kubevirt_web_ui/defaults/main.yml

```diff
@@ -1,23 +1,14 @@
 ---
-# Required param:
-# - kubevirt_web_ui_image_name
-#
-# TODO: gather automatically
-# - openshift_master_default_subdomain
-# - public_master_hostname
+openshift_client_binary: "oc"
```

Comment:
why make this a var?

Reply:
The line has been moved from bellow, this change is left-over from development of this PR - not needed anymore but causing no harm.
The variable itself is taken from the original openshift-ansible, kept here for consistency between these two projects. Considering rest of the kubevirt-ansible code, it has no benefit since the `oc` is referenced here directly.